src/main/java/reposense/parser/RepoConfigCsvParser.java

```diff
@@ -42,6 +44,21 @@ public RepoConfigCsvParser(Path csvFilePath) throws IOException {
         };
     }
 
+    /**
+     * Parses the csv file associated with this instance of the {@code CsvParser} and returns a {@code List}
+     * containing the RepoConfigurations in this file.
+     *
+     * @throws IOException if there are 0 valid records in the csv, or if there are errors accessing the file.
```

Comment:
@throws ... if there are **no** valid records ...

Reply:
Thanks. I will update this comment.

Reply:
Done.